adapter/internal/management-server/xds/client.go

```diff
@@ -135,6 +169,30 @@ func watchApplications() {
 	}
 }
 
+func watchSubscriptions() {
+	for {
+		discoveryResponse, err := xdsSubStream.Recv()
+		if err == io.EOF {
+			loggers.LoggerXds.ErrorC(logging.GetErrorByCode(1702, err.Error()))
```

Comment:
i couldn't find error codes 1702, 1703 in error list, did you defined them?

Reply:
I do not see that you have defined any of the error codes added by this PR. Please fix it

Reply:
fixed and resolved